tests/unit/Services/ReapplyServiceTest.php

```
@@ -227,6 +342,47 @@ public function testHashFailureBeforeWriteIsReported(): void
+         * file (unchanged) gets reapplied, proving the loop kept iterating after the first skip.
+         */
+        $this->seedTracked('first.txt', "stub\n", "stub\n");
+        $this->seedTracked('second.txt', "stub\n", "stub\n");
+
+        // target the first destination exclusively so the mock does not leak into the second iteration's hash calls.
+        MockerState::addCondition(
+            'yii\\scaffold\\Scaffold\\Lock',
+            'hash_file',
+            ['sha256', "{$this->tempDir}/first.txt", false, []],
+            false,
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xepozz\InternalMocker\MockerState' in method 'testHashFailureSkipContinuesIterationToNextReapplicableEntry'. (undefined)

(StaticAccess)